projects/common/src/main/java/dan200/computercraft/shared/common/AbstractContainerBlockEntity.java

```diff
@@ -26,9 +23,4 @@ protected AbstractContainerBlockEntity(BlockEntityType<?> type, BlockPos pos, Bl
     protected final Component getDefaultName() {
         return Component.translatable(getBlockState().getBlock().getDescriptionId());
     }
-
-    @Override
-    public boolean stillValid(Player player) {
-        return BlockEntityHelpers.isUsable(this, player, BlockEntityHelpers.DEFAULT_INTERACT_RANGE);
```

Comment:
Reminder to double-check why we removed this. Possible it's just implemented in `BaseContainerBlockEntity` now? Is this really not the case in 1.20.1?